src/main/java/models/Notification.java

```diff
@@ -4,30 +4,36 @@
  * Holder model for the notification
  */
 public class Notification {
-    private int id;
-    private String text, link, timeStamp;
+    private String notificationText, link;
+    private long timeStamp;
+    private int userId;
     private boolean read;
-    Notification(int id, String text, String link, String timeStamp, boolean read) {
-        this.id = id;
-        this.text = text;
+    public Notification(String text, String link, long timeStamp, boolean read) {
```

Comment:
I think this constructor will not be used because arangodb instance will use empty constructor to initialize a Notification instance. So can you replace it with setter methods?

Reply:
It will be used in `SendNotificationCommand` because this command will instantiate a new `Notification` instance with the attributes extracted from the HashMap. It will be more readable to pass the attributes to a constructor rather than using setter methods

Reply:
I agree but do you grantee that the same arguments will be sent ? If yes then you are right :+1:

Reply:
Yes each notification should have the same arguments everytime